app/src/main/AndroidManifest.xml

```diff
@@ -9,8 +9,10 @@
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
+        android:requestLegacyExternalStorage="true"
         android:theme="@style/AppTheme">
-        <activity android:name=".MainActivity">
+        <activity android:name=".MainActivity"
+                  android:exported="true">
```

Comment:
Why do we need the exported flag?

Reply:
This flag is mandatory to build the app in Android 12+